packages/flutter/test/cupertino/route_test.dart

```diff
@@ -2690,6 +2690,202 @@ void main() {
     await tester.tap(find.text('tap'));
     await tester.pumpAndSettle();
   });
+
+  testWidgets('Setting CupertinoDialogRoute.requestFocus to false does not request focus on the dialog', (WidgetTester tester) async {
+    late BuildContext savedContext;
+    final FocusNode focusNode = FocusNode();
+    addTearDown(focusNode.dispose);
+    const String dialogText = 'Dialog Text';
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Builder(
+          builder: (BuildContext context) {
+            savedContext = context;
+            return CupertinoTextField(focusNode: focusNode);
+          },
+        ),
+      ),
+    );
+    await tester.pump();
+
+    FocusNode? getCupertinoTextFieldFocusNode() {
+      return tester.widget<Focus>(find.descendant(
+        of: find.byType(CupertinoTextField),
+        matching: find.byType(Focus),
+      )).focusNode;
+    }
```

Comment:
Nit: Would there be any benefit to reusing this instead of declaring it twice?

Reply:
This is declared once but reused 5 times actually.
<img width="361" alt="image" src="https://github.com/user-attachments/assets/a972d66d-1137-40dd-b8c2-f23166a8fe9d">